### PR TITLE
Feature: Create payout mutation

### PR DESCRIPTION
**Pull request by rasmusguterstam**
# Jira Issues: [IP-15](https://hedvig.atlassian.net/browse/IP-15) & [IP-16](https://hedvig.atlassian.net/browse/IP-15) 

## What?
- Implement payout mutation 
- Kill off redux code

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [x] Codescouted
- [ ] Unit tests written
- [x] Tested locally

### CODE REVIEW

**Review by joacimastrom — APPROVED** [review]
⭐ ⭐

File: src/components/payouts/payout-details/index.tsx [hunk 1]
```diff
@@ -46,22 +43,44 @@ const getPayoutValidationSchema = () =>
...
+        toast.promise(
+          payoutMember({
+            variables: {
+              memberId,
+              request: {
+                amount: {
+                  amount: data.amount,
+                  currency: 'SEK',
+                },
+                category: data.category,
```
[the remark this answers is not in the document]
Comment (joacimastrom) [reply src/components/payouts/payout-details/index.tsx:68]: I guess the way to go would be to define an interface for the data up on row 54, then you could do whatever you want here :)

**Review by vonElfvin — COMMENTED** [review]
This is sooooo nice! 💯 This PR and the authentication one will be a new chapter for `hedgehog` 😌  I do have one concern though that I think we should address.

File: src/components/payouts/payout-details/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (rasmusguterstam) [reply src/components/payouts/payout-details/index.tsx:68]: I realized now that the problem here is that the form is using a slight hack right now, and I'd say this entire form needs refactoring! Will do in a future push on killing Formik

File: src/components/payouts/payout-details/index.tsx [hunk 2]
```diff
@@ -46,22 +43,44 @@ const getPayoutValidationSchema = () =>
...
-        setConfirmed(false)
+
+        toast.promise(
+          payoutMember({
+            variables: {
+              memberId,
+              request: {
+                amount: {
+                  amount: data.amount,
+                  currency: 'SEK',
```
[the remark this answers is not in the document]
Comment (rasmusguterstam) [reply src/components/payouts/payout-details/index.tsx:66]: Yeah it was hardcoded before, and I believe the reason was that payouts only are available in sweden

**Pull request merged**